src/test/resources/peer-review/peer-review.pone.0207232.xml

```diff
@@ -17,6 +17,13 @@
             </name>
             <role>Academic Editor</role>
           </contrib>
+          <contrib contrib-type="author">
+            <name name-style="western">
```

Comment:
From a brief look at the the other XSLT it seems we don't support other name-styles. Do we know that this is the case?

Reply:
I don't recall asking anybody but I remember coming to the conclusion with Matt after reading the explicit comments in the article transform, as you likely observed:

`<!-- commented out eastern/western logic. edit when we implement name-style eastern (possibly use generic nlm) -->`  

Do you want me to look into it further?

Reply:
No, if you and Matt had the same conclusion I did then :+1: